### ISSUE TITLE
open jit in a platform_config_file seems nosense

### ISSUE THREAD

**Issue by venuswu** [tensorflow/serving#1515]
```
import tensorflow as tf
from tensorflow_serving.config import platform_config_pb2
from tensorflow_serving.servables.tensorflow import session_bundle_config_pb2
from tensorflow_serving.servables.tensorflow import saved_model_bundle_source_adapter_pb2

session_config = tf.compat.v1.ConfigProto()
session_config.gpu_options.allow_growth = True
session_config.gpu_options.per_process_gpu_memory_fraction = 0.4
session_config.graph_options.optimizer_options.global_jit_level = tf.compat.v1.OptimizerOptions.ON_1

legacy_config=session_bundle_config_pb2.SessionBundleConfig(session_config=session_config)
adapter = saved_model_bundle_source_adapter_pb2.SavedModelBundleSourceAdapterConfig(legacy_config=legacy_config)

config_map = platform_config_pb2.PlatformConfigMap()
config_map.platform_configs['tensorflow'].source_adapter_config.Pack(adapter)
print(config_map)
```

I wrote the code as above, and run a tf-serving with the output, but the xla-jit seems still not open.
Can the jit not be used in tf-serving ??

**Comment by gowthamkpr** [comment]
I am unclear as to what your issue is @venuswu Can you please follow the template https://github.com/tensorflow/serving/issues/new[here](url) and be more clear in description. Thanks!

**Comment by venuswu** [comment]
I want to know whether xla-jit can be used in tf-serving ?

**Comment by gowthamkpr** [comment]
We actually suggest against doing compilations during inference. Compilations at inference time will cause HBM to be out of memory, resulting the chips to be unable to serve requests. Please see [here](http://go/servo-faq#i-am-getting-genericresource-exhausted-error-in-my-model-server-how-can-i-fix-it) for more details.

**Comment by venuswu** [comment]
Does the same problem of HBM happen during traing? @gowthamkpr

**Comment by gowthamkpr** [comment]
no it doesn't happen. It only happens during inference

**Comment by venuswu** [comment]
May I ask why it only happens during inference ?

**Comment by gowthamkpr** [comment]
@venuswu You can refer to the following [question](https://stackoverflow.com/questions/54671395/tensorflow-serving-with-xla) for more explanation. Its mostly due to high latency for compilation.

**Comment by gowthamkpr** [comment]
I am going to close this issue as it has been answered. Please add additional comments and we can open this issue again. Thanks!

**Comment by jeisinge** [comment]
To confirm, XLA cannot be used with TensorFlow Serving?

**Comment by venuswu** [comment]
It can be used, but the XLA is influenced by the shape of the input tensor, so batching is not so efficient under inference with XLA. @jeisinge

**Comment by jeisinge** [comment]
FYI, we tested TF-Serving and Triton Inference Server.  We found that Triton had an option that enabled XLA which gave us our best performance.

https://docs.nvidia.com/deeplearning/triton-inference-server/user-guide/docs/protobuf_api/model_config.proto.html#_CPPv4N9inference23ModelOptimizationPolicy5GraphE
> For TensorFlow causes XLA to be enabled/disabled for the model.

I don't know if enabling JIT means enabling XLA.
